Dockerfiles/Dockerfile.bcda

```diff
@@ -7,6 +7,7 @@ RUN go get github.com/tools/godep
 RUN go get github.com/pressly/fresh
 RUN go get -u github.com/derekparker/delve/cmd/dlv
 
+
```

Comment:
A couple extraneous newlines in the Dockerfiles.

Reply:
removed in next commit.

Reply:
removed